src/shared/scrapers/USA/MA/index.js

```diff
@@ -132,6 +132,10 @@ const scraper = {
       let sumDeaths = 0;
 
       data.features.forEach(item => {
+        if (item.attributes.County === 'TOTALS') {
+          return;
```

Comment:
If we're including hospitalization too now, then you can get that from the totals row and then return. But that'd be a future change.